tools/check_server_cert.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+c_red="\033[0;31m"
+c_green="\033[0;32m"
+c_yellow="\033[0;33m"
+c_reset="\033[0m"
+
+success() {
+        echo -e "${c_green}$1${c_reset}"
+}
+
+warning() {
+        echo -e "${c_yellow}$1${c_reset}"
+}
+
+error() {
+        echo -e "${c_red}$1${c_reset}"
+}
+
+die() {
+    >&2 echo -e "${c_red}$*${c_reset}"
+    exit 1
+}
+
+CONFIG=$1
+
+if [ ! -f "$CONFIG" ]; then
+  die "could not find json config file: $CONFIG"
+fi
+
+HOSTNAME=$(jq -r ".hostname" "$CONFIG")
+CA=$(jq -r ".ca" "$CONFIG")
```

Comment:
Should this be `.ca_file`?
https://github.com/teslamotors/fleet-telemetry/blob/613041e96d20ceba01a731fe2d42d4e0b5c6440e/config/config.go#L131

```suggestion
CA=$(jq -r ".ca_file" "$CONFIG")
```

Reply:
Ohhhh. Okay, the new README updates make this make sense, then! Thanks!